### ISSUE TITLE
Preselect options

### ISSUE THREAD

**Issue by fgmacska** [basvandenberg/ng-select#114]
Hi,

How to preselect options ? With select method ? It sounds easy to do but I don't success.

**Comment by matthewhegarty** [comment]
Does [this plunkr](https://plnkr.co/edit/UEHvUHyzOakZYL3Sdltw?p=preview) help?
If not, can you provide code example of what is not working?

**Comment by basvandenberg** [comment]
I hope @matthewhegarty plunkr helped, also have a look at the [examples](https://basvandenberg.github.io/ng-select/). Please use stackoverflow for questions and only post issues here.

**Comment by Satawe-no** [comment]
Hi, ok @basvandenberg I will know for the next time, and thank you for the examples.

@matthewhegarty Yes, your code helped me to understand that my mistake was not come from my ng-select code. Actually by default, for the value, I tried to use an object, but we have to use string to make things works. I've also made scope mistake and finally the data didn't load before rendering the component. So I've used "Resolve" to pre-fetch data before activating the route.

Thanks again.

**Comment by fgmacska** [comment]
Hi, ok @basvandenberg I will know for the next time, and thank you for the examples.

@matthewhegarty Yes, your code helped me to understand that my mistake was not come from my ng-select code. Actually by default, for the value, I tried to use an object, but we have to use string to make things works. I've also made scope mistake and finally the data didn't load before rendering the component. So I've used "Resolve" to pre-fetch data before activating the route.

Thanks again.
